packages/core-data/package.json

```diff
@@ -24,6 +24,8 @@
 	"dependencies": {
 		"@babel/runtime": "^7.4.4",
 		"@wordpress/api-fetch": "file:../api-fetch",
+		"@wordpress/block-editor": "file:../block-editor",
```

Comment:
It seems core-data does not imports anything from @wordpress/block-editor, what is the reason for specifying this dependency?

Reply:
Must have mushed that in from another branch. Removed 👍